src/annotator/test/guest-test.js

```diff
@@ -487,6 +492,22 @@ describe('Guest', () => {
         assert.notCalled(guest.crossframe.call);
       }
     });
+
+    it('reposition the adder on window "resize" event', () => {
+      sandbox.stub(guest, '_repositionAdder').callThrough();
+      sandbox.stub(guest, '_onSelection');
+      window.dispatchEvent(new Event('resize'));
+
+      // If the adder is not displayed on 'window.resize' event does nothing
```

Comment:
Maybe reword this a little...

"Does not call _onSelection() if the adder is hidden"

Reply:
Good idea